README.md

```diff
@@ -59,7 +59,7 @@ In order for Thimble to be installed correctly, the following dependencies need
 - Virtualbox (version 5.1 or later) [[download](https://www.virtualbox.org/wiki/Downloads)]
 - Vagrant (version 1.9 or later) [[download](https://www.vagrantup.com/downloads.html)]
   - __Note:__ On Windows machines, you may need to restart your computer after installing Vagrant for it to be fully usable.  
-  Avoid installation in directories containing *spaces*.  Vagrant is written in Ruby which has issues with directory names containing spaces, so be sure that your VAGRANT_HOME environment variable does not contain any spaces (i.e. a user home folder w/ spaces).  You can set VAGRANT_HOME via Control Panel > Advanced > Environment variables or set it from command prompt with [setx](https://technet.microsoft.com/en-us/library/cc755104(v=ws.11).aspx) command.  
+  Avoid installation in directories containing *spaces*.  Vagrant is written in Ruby which has issues with directory names containing spaces, so be sure that your VAGRANT_HOME environment variable does not contain any spaces (i.e. a user home folder w/ spaces).  You can set VAGRANT_HOME via Control Panel > Advanced system settings > Environment variables > System Variables or set it from command prompt with [setx](https://technet.microsoft.com/en-us/library/cc755104(v=ws.11).aspx) command `setx VAGRANT_HOME c:\.vagrant.d -m`.
```

Comment:
markdown parsers don't gracefully deal with parentheses inside the `[]()` pattern, so you'll want to swap those out with the corresponding URL-encoded `%28` and `%29`